rn/Teacher/src/modules/speedgrader/__tests__/SpeedGrader.test.js

```diff
@@ -83,11 +84,13 @@ let defaultProps = {
   assignmentSubmissionTypes: ['none'],
   gradeSubmissionWithRubric: jest.fn(),
   getCourseEnabledFeatures: jest.fn(),
+  getEnabledFeatureFlags: jest.fn(() => Promise.resolve({ data: [] })),
 }
 
 describe('SpeedGrader', () => {
   beforeEach(() => {
     jest.clearAllMocks()
+    SpeedGrader.drawerState = new DrawerState()
```

Comment:
Reseting the `drawerState` on each test is what caused the giant change in the snapshot.